pkg/plugins/resources/k8s/native/api/v1alpha1/mesh_gateway_instance.go

```diff
@@ -30,8 +30,7 @@ type MeshGatewayInstanceSpec struct {
 	// since is auto-generated, and should match exactly one Gateway
 	// resource.
 	//
-	// +required
-	// +kubebuilder:validation:MinLen=1
+	// +optional
```

Comment:
Do you need to remove the minlen validation?

Reply:
We don't need, reverted